packages/ibm-products/src/components/ConditionBuilder/ConditionBuilder.js

```diff
@@ -95,7 +96,10 @@ export let ConditionBuilder = React.forwardRef(
           ref={conditionBuilderRef}
           {...getDevtoolsProps(componentName)}
         >
-          <VStack className={`${blockClass}__${variant}`}>
+          <VStack
+            className={`${blockClass}__${variant}`}
+            onKeyDown={(e) => handleKeyDown(e, conditionBuilderRef)}
```

Comment:
minor, but lets move this out of the html. i think it improves readability to remove as much non-html as possible. so something like this would look better
```
const keyPressHandler = (evt) => { ... };
...
onKeyDown(keyPressHandler)
```